src/repositories/tweet.repository.ts

```diff
@@ -0,0 +1,44 @@
+import { ObjectId } from "mongodb";
+
+import { TweetModel } from "../models/Tweet";
+
+export const getOneById = (id: string) => {
+  return TweetModel.find({ _id: id });
+};
+export const createOne = async (
+  authorId: string,
+  content: string,
+  repliedTo?: string
+) => {
+  const tweetModel = new TweetModel({
+    authorId: new ObjectId(authorId),
+    content: content,
+    repliedTo: new ObjectId(repliedTo),
```

Comment:
```suggestion
    authorId: authorId,
    content: content,
    repliedTo: repliedTo,
```
use string, Mongoose converts it under hood
[see example](https://github.com/attech-org/leleka-backend/blob/develop/src/repositories/bookmarks.repository.ts)